### PR TITLE
Fix bugs caught on QA

### PR DESCRIPTION
**Pull request by MounirDhahri**
The type of this PR is: **Bugfix**

This PR resolves [CX-1358]

### Description
- Fix auction results filter modal title
Group 1 (6)

___
- Fix palette button loading state
**Before**

https://user-images.githubusercontent.com/11945712/116124643-d73f9000-a6c4-11eb-9887-20299a391bf6.mov

**After**

https://user-images.githubusercontent.com/11945712/116124634-d4dd3600-a6c4-11eb-88f1-665bcae53014.mov

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-434]: https://artsyproduct.atlassian.net/browse/CX-434
[CX-1358]: https://artsyproduct.atlassian.net/browse/CX-1358

### CODE REVIEW

File: src/palette/elements/Button/Button.tsx [hunk 1]
```diff
@@ -287,20 +289,16 @@ export const Button: React.FC<ButtonProps> = (props) => {
...
-                  <Spinner size={size} color={spinnerColor} />
-                )}
+                <VisibleTextContainer>
+                  <Sans weight="medium" color={loadingStyles.textColor || to.textColor} size={s.size}>
+                    {children}
+                  </Sans>
+                </VisibleTextContainer>
+                <HiddenText role="presentation" weight="medium" size={s.size}>
+                  {longestText ? longestText : children}
+                </HiddenText>
```
Comment (MounirDhahri) [inline src/palette/elements/Button/Button.tsx:299]: I reverted this to be as it was before my changes in last week's PR

File: src/palette/elements/Button/Button.tsx [hunk 2]
```diff
@@ -212,13 +212,15 @@ export const Button: React.FC<ButtonProps> = (props) => {
         backgroundColor: `rgba(0, 0, 0, ${opacity})`,
         color: color("white100"),
         borderWidth: 0,
+        textColor: "transparent",
```
Comment (pvinis) [inline src/palette/elements/Button/Button.tsx:215]: oof. dirty but nice one.

i look forward to a reworking of this component haha.

File: src/lib/Components/ArtworkFilter/ArtworkFilterOptionsScreen.tsx [hunk 3]
```diff
@@ -114,17 +114,13 @@ export const ArtworkFilterOptionsScreen: React.FC<
   return (
     <Flex style={{ flex: 1 }}>
       <Flex flexGrow={0} flexDirection="row" justifyContent="space-between" alignItems="center" height={space(6)}>
-        <Flex flex={1} alignItems="flex-start">
+        <Flex position="absolute" alignItems="flex-start">
```
Comment (pvinis) [inline src/lib/Components/ArtworkFilter/ArtworkFilterOptionsScreen.tsx:117]: didn't we fix this in the android bug bash already? 🤔

**Review by pvinis — APPROVED** [review]

File: src/lib/Components/ArtworkFilter/ArtworkFilterOptionsScreen.tsx (hunk 3 above)
Comment (MounirDhahri) [reply src/lib/Components/ArtworkFilter/ArtworkFilterOptionsScreen.tsx:117]: Yeah we set it for the `FancyModalHeader` and I did the same here

File: src/palette/elements/Button/Button.tsx (hunk 2 above)
Comment (MounirDhahri) [reply src/palette/elements/Button/Button.tsx:215]: yeah, it was weird for me as well but didn't want to make big changes before submitting the app

File: src/lib/Components/ArtworkFilter/ArtworkFilterOptionsScreen.tsx (hunk 3 above)
Comment (pvinis) [reply src/lib/Components/ArtworkFilter/ArtworkFilterOptionsScreen.tsx:117]: Ah, gotcha. I thought it was the same file. Thanks.

**Pull request merged**